redisinsight/api/src/modules/auth/microsoft-auth/microsoft-azure-auth.module.ts

```diff
@@ -1,9 +1,14 @@
 import { Module } from '@nestjs/common';
 import { MicrosoftAuthService } from './microsoft-azure-auth.service';
 import { MicrosoftAzureAuthController } from './microsoft-azure-auth.controller';
+import { CloudSessionModule } from 'src/modules/cloud/session/cloud-session.module';
+import { MicrosoftAuthStorageModule } from './microsoft-auth-storage.module';
 
 @Module({
-    imports: [],
+    imports: [
+        CloudSessionModule.register(),
```

Comment:
will this create new instance of cloudSessionModule with all default dependencies?